openmc/data/angle_distribution.py

```diff
@@ -32,6 +32,16 @@ def __init__(self, energy, mu):
         self.energy = energy
         self.mu = mu
 
+    def __eq__(self, other):
+        if not isinstance(other, type(self)):
+            return NotImplemented
+        else:
+            eqval = True
+            if (not np.array_equal(self.energy, other.energy) or
+                not np.array_equal(self.mu, other.mu)):
```

Comment:
It will work; `Tabular` has an `__eq__` implementation which array_equal calls for the above comparison.  In fact the script I wrote to look at differences between ACE files at a given temperature is doing just that and has correctly compared lists of two `Tabular` objects (like mu above).  

`Legendre` also has an `__eq__` implementation, but comparing a `Tabular` to `Legendre` will return NotImplemented because of the type comparison check.